messagebird/client.py

```diff
@@ -1,187 +1,255 @@
+
+    def lookup_hlr(self, phonenumber, params=None):
+        """Retrieve the information of a specific HLR lookup."""
+        if params is None: params = {}
+        return HLR().load(self.request('lookup/' + str(phonenumber) + '/hlr', 'GET', params))
+
+    def lookup_hlr_create(self, phonenumber, params=None):
+        """Perform a new HLR lookup."""
+        if params is None: params = {}
+        return HLR().load(self.request('lookup/' + str(phonenumber) + '/hlr', 'POST', params))
+
+    def verify(self, id):
+        """Retrieve the information of a specific verification."""
+        return Verify().load(self.request('verify/' + str(id)))
+
+    def verify_create(self, recipient, params=None):
+        """Create a new verification."""
+        if params is None: params = {}
+        params.update({'recipient': recipient})
+        return Verify().load(self.request('verify', 'POST', params))
+
+    def verify_verify(self, id, token):
+        """Verify the token of a specific verification."""
+        return Verify().load(self.request('verify/' + str(id), params={'token': token}))
+
+    def contact(self, id):
+        """Retrieve the information of a specific contact."""
+        return Contact().load(self.request('contacts/' + str(id)))
+
+    def contact_create(self, phonenumber, params=None):
+        if params is None: params = {}
+        params.update({'msisdn': phonenumber})
+        return Contact().load(self.request('contacts', 'POST', params))
+
+    def contact_delete(self, id):
+        self.request_plain_text('contacts/' + str(id), 'DELETE')
+
+    def contact_update(self, id, params=None):
+        self.request_plain_text('contacts/' + str(id), 'PATCH', params)
+
+    def contact_list(self, limit=10, offset=0):
+        query = self._format_query(limit, offset)
+        return ContactList().load(self.request('contacts?' + query, 'GET', None))
+
+    def group(self, id):
+        return Group().load(self.request('groups/' + str(id), 'GET', None))
+
+    def group_create(self, name, params=None):
+        if params is None: params = {}
+        params.update({'name': name})
+        return Group().load(self.request('groups', 'POST', params))
+
+    def group_delete(self, id):
+        self.request_plain_text('groups/' + str(id), 'DELETE', None)
+
+    def group_list(self, limit=10, offset=0):
+        query = self._format_query(limit, offset)
+        return GroupList().load(self.request('groups?' + query, 'GET', None))
+
+    def group_update(self, id, name, params=None):
+        if params is None: params = {}
+        params.update({'name': name})
+        self.request_plain_text('groups/' + str(id), 'PATCH', params)
+
+    def group_add_contacts(self, groupId, contactIds):
+        query = self.__group_add_contacts_query(contactIds)
+        self.request_plain_text('groups/' + str(groupId) + '?' + query, 'PUT', None)
+
+    def __group_add_contacts_query(self, contactIds):
```

Comment:
Is double underscore made here for a purpose?

Reply:
this is in the code base already, I just reformatted that